consul_exporter.go

```diff
@@ -86,7 +90,18 @@ func NewExporter(uri string) *Exporter {
 			[]string{"service", "node"},
 		),
 
+		keyValues: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "catalog_kv",
+				Help:      "key/value",
```

Comment:
Please add a more verbose help string for this metric. I guess there can be values that are not numeric and those will be skipped completely, so that's at least one important thing to mention here.

How about:

"The values for selected keys in Consul's key/value catalog. Keys with non-numeric values are omitted."